server/config/tagType/tagType.js

```diff
@@ -0,0 +1,9 @@
+const config = require('./tagConfig.json');
+
+const fetchTagType = (name) => {
+    return config[name] ?? [];
```

Comment:
As this is a direct lookup per tag, this probably should return an object and not an array?